### DISCUSSION TITLE
Spare NanoELS PCBs (to a good home)

### DISCUSSION THREAD

**Discussion by mockendon** [kachurovskiy/nanoels#28] [General]
Fellow makers, I have 4 spare v1 PCBs from JLCPCB.  If you would like to pay the postage (from the UK) I will send you an early (or late) xmas present.  First come first served - better than them going in the pile in the workshop!  I could also build up a board if that would help someone out as have a fair few spare parts too (for the price of a coffee).

Matt

**Comment by bestbean** [comment]
Here!

**Reply by mockendon** [reply]
I’m sure Github used to have a PM service.  🙄 email me your details then - github@ockendon.net the Royal Mail should have it to you by about Easter 😬

**Comment by bestbean** [comment]
Thanks for this offer!
Meanwhile, i got 5 new PCBs, 3 of them already given away.
